tools/java/org/unicode/cldr/tool/CountItems.java

```diff
@@ -85,35 +85,36 @@
         + " America/Los_Angeles America/Phoenix America/Denver America/Chicago America/Indianapolis"
         + " America/New_York";
 
-    static final Map<String, String> country_map = CollectionUtilities.asMap(new String[][] {
-        { "AQ", "http://www.worldtimezone.com/time-antarctica24.php" },
-        { "AR", "http://www.worldtimezone.com/time-south-america24.php" },
-        { "AU", "http://www.worldtimezone.com/time-australia24.php" },
-        { "BR", "http://www.worldtimezone.com/time-south-america24.php" },
-        { "CA", "http://www.worldtimezone.com/time-canada24.php" },
-        { "CD", "http://www.worldtimezone.com/time-africa24.php" },
-        { "CL", "http://www.worldtimezone.com/time-south-america24.php" },
-        { "CN", "http://www.worldtimezone.com/time-cis24.php" },
-        { "EC", "http://www.worldtimezone.com/time-south-america24.php" },
-        { "ES", "http://www.worldtimezone.com/time-europe24.php" },
-        { "FM", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "GL", "http://www.worldtimezone.com/index24.php" },
-        { "ID", "http://www.worldtimezone.com/time-asia24.php" },
-        { "KI", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "KZ", "http://www.worldtimezone.com/time-cis24.php" },
-        { "MH", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "MN", "http://www.worldtimezone.com/time-cis24.php" },
-        { "MX", "http://www.worldtimezone.com/index24.php" },
-        { "MY", "http://www.worldtimezone.com/time-asia24.php" },
-        { "NZ", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "PF", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "PT", "http://www.worldtimezone.com/time-europe24.php" },
-        { "RU", "http://www.worldtimezone.com/time-russia24.php" },
-        { "SJ", "http://www.worldtimezone.com/index24.php" },
-        { "UA", "http://www.worldtimezone.com/time-cis24.php" },
-        { "UM", "http://www.worldtimezone.com/time-oceania24.php" },
-        { "US", "http://www.worldtimezone.com/time-usa24.php" },
-        { "UZ", "http://www.worldtimezone.com/time-cis24.php" }, });
+    static final ImmutableMap<String, String> country_map = ImmutableMap.<String, String>builder()
```

Comment:
These are the "asMap()" changes which were manual (but still mostly just search & replace).